examples/books_bloc/lib/main.dart

```diff
@@ -14,10 +14,9 @@ class HomeScreen extends StatelessWidget {
       ),
       body: Center(
         child: ElevatedButton(
-          onPressed: () {
-            context.beamToNamed('/books');
-            context.read<BooksBloc>().add(LoadBooks());
-          },
+          onPressed: () => context
+            ..read<BooksBloc>().add(LoadBooks())
+            ..beamToNamed('/books'),
```

Comment:
This is a very cool place to use cascade operator. Feels so right.

Reply:
> This is a very cool place to use cascade operator. Feels so right.

☺️